djstripe/admin.py

```diff
@@ -12,8 +11,7 @@
 
 from django.contrib import admin
 
-from .models import Event, EventProcessingException, Transfer, Charge, Plan
-from .models import Invoice, InvoiceItem, Subscription, Customer
+from . import models
```

Comment:
our convention is to explicitly implement each class. Those two lines do need to be combined though.